config/default/phpmd.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<ruleset name="Youwe/Global"
+         xmlns="http://pmd.sf.net/ruleset/1.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 https://pmd.sourceforge.io/ruleset_xml_schema.xsd">
+    <description>
```

Comment:
This looks a bit odd, why not like this?
<description> Global defined PHPMD rules </description>